docusaurus/docs/create-a-plugin/develop-a-plugin/manage-go.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: backend
-title: Backend
+id: manage-go
+title: Manage Go for backend
```

Comment:
This title doesn't make sense for me, we're not managing Go :) 

You could mirror the frontend title as an option.

Reply:
Done in companion PR.